CMakeLists_files.cmake

```diff
@@ -31,6 +31,10 @@
 list (APPEND TEST_SOURCE_FILES
 	)
 
+list(APPEND MAIN_SOURCE_FILES
+	opm/upscaling/RelPermUtils.cpp
```

Comment:
It's fine with me. I do think that until we decide on a policy for private headers we should default to making them public though, especially in the more upstream modules.